packages/bspwm/fix-hardcoded-path.patch

```diff
@@ -0,0 +1,69 @@
+diff -uNr bspwm-0.9.10/doc/bspwm.1 bspwm-0.9.10.mod/doc/bspwm.1
+--- bspwm-0.9.10/doc/bspwm.1	2020-08-02 14:51:32.000000000 +0700
++++ bspwm-0.9.10.mod/doc/bspwm.1	2022-03-22 15:59:57.637000678 +0700
+@@ -1606,7 +1606,7 @@
+ \fBbspc\fR
+ and
+ \fBbspwm\fR\&. If it isn\(cqt defined, then the following path is used:
+-\fI/tmp/bspwm<host_name>_<display_number>_<screen_number>\-socket\fR\&.
++\fI@TERMUX_PREFIX@/tmp/bspwm<host_name>_<display_number>_<screen_number>\-socket\fR\&.
```

Comment:
Seeing this patch where you are fixing hard-coded path forma pages. We do have a lot in other manpages too. Also in some of popularly used packages. Seems good though